packages/cli/src/net/createConjureContext.mts

```diff
@@ -18,12 +18,13 @@ import type { ConjureContext } from "conjure-lite";
 import { createFetch } from "./createFetch.mjs";
 
 export function createConjureContext(
-  baseUrl: string,
+  foundryUrl: string,
   servicePath: "/artifacts/api",
+  token: string,
```

Comment:
For all of the client layer code let's keep it consistent in accepting a token provider rather than a static token string. In the CLI code itself it's fine that the token is loaded once from the cascading logic as the command is short lived but the clients generally may want to have a longer lived context that can have the token swapped out under the hood